nixos/modules/services/monitoring/cadvisor.nix

```diff
@@ -84,6 +84,16 @@ in {
         type = types.bool;
         description = "Cadvisor storage driver, enable secure communication.";
       };
+
+      extraOptions = mkOption {
+        type = types.listOf types.str;
+        default = [];
+        description = ''
+          Additional cadvisor options.
+          
+          See https://github.com/google/cadvisor/blob/master/docs/runtime_options.md for available options.
```

Comment:
Needs the <link xlink:href="https://example.com"/> incantation

Reply:
If you are referring to the xml link, I intentionally didn't use that because they used to (still do?) vanish in the manpage rendering.

Reply:
Haha, it vanished from my comment because I didn't quote it, I also wrote `<link xlink:href="https://example.com"/>`

Reply:
And I think only `<link xlink:href="https://example.com/">Example</link>` ones vanish from the man pages